projects/storefrontapp-e2e-cypress/cypress/e2e/regression/user_access/navigation-login-a11y.e2e.cy.ts

```diff
@@ -0,0 +1,51 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as login from '../../../helpers/login';
+import { viewportContext } from '../../../helpers/viewport-context';
+
+function assertNavigationButtonsAttributes(buttonsSelector: string) {
+  cy.get(buttonsSelector).each(($btn) => {
+    const btnAriaLabel = $btn.attr('aria-label');
+    cy.wrap($btn)
+      .should('have.attr', 'title', `${btnAriaLabel} Menu`)
+      .should('have.attr', 'aria-label', btnAriaLabel)
+      .should('have.attr', 'aria-controls', btnAriaLabel);
+  });
+}
+
+describe('Navigation Login', () => {
+  let user;
+  viewportContext(['desktop'], () => {
+    before(() => {
+      cy.visit('/login');
+      user = login.registerUserFromLoginPage();
+    });
+
+    it('should login and logout successfully and have correct Navigation Menu buttons values', () => {
+      login.loginUser();
+
+      const tokenRevocationRequestAlias =
+        login.listenForTokenRevocationRequest();
+
+      const profileMenuRootBtnSelector =
+        'cx-login > cx-page-slot nav ul li:first-child button';
+      cy.get(profileMenuRootBtnSelector)
+        .contains('My Account')
+        .invoke('attr', 'ariaLabel')
+        .contains(`Hi, ${user.firstName} ${user.lastName}`);
```

Comment:
We can simplify this with shorter selectors and aliases. We should try and follow thy Cypress way. 
```suggestion
      cy.get('cx-login button')
        .as('myAccountBtn')
        .invoke('attr', 'ariaLabel')
        .contains(`Hi, ${user.firstName} ${user.lastName}`);
        // ....
        cy.get('@myAccountBtn').should('not.exist');
```

Reply:
OK thanks, i have applied the changes.

Reply:
@Pio-Bar it should be now addressed. Could you please dbl check?